Makefile

```diff
@@ -43,6 +39,13 @@ IMAGE_REPO ?= quay.io/shipwright
 TAG ?= latest
 IMAGE_PUSH ?= true
 
+# Image URL to use all building/pushing image targets
+IMG ?= $(IMAGE_REPO)/shipwright-operator:$(VERSION)
+
+# BUNDLE_IMG defines the image:tag used for the bundle. 
+# You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
+BUNDLE_IMG ?= $(IMAGE_REPO)/shipwright-operator-bundle:$(VERSION)
```

Comment:
Why rename to `shipwright-operator-bundle`? I think `quay.io/shipwright/operator-bundle` is a fine image reference.

Reply:
well, all the `operator-sdk` documentation goes with this naming, `<product>-operator-bundle` and `<product>-operator`. that is why i changed it to look this way.
with our registry it is also going to be okay, `quay.io/shipwright/operator` but with any other registry it might end up being undistinguishable, like `some-registry.io/user/operator`.
let's discuss it quickly today, i'll change it if necessary